lib/rabbitmq/cli/plugins/commands/list_command.ex

```diff
@@ -114,6 +114,23 @@ defmodule RabbitMQ.CLI.Plugins.Commands.ListCommand do
       plugins: format_plugins(plugins, format, enabled, enabled_implicitly, running)}
   end
 
+  defp remote_running_plugins(node) do
```

Comment:
`rabbitmq-server` implementation was split into 2 rpc calls only for compatibility. Maybe it's better to add new function to `rabbit_plugins` that'll return a list of running plugins already augment with versions - as there is nothing yet to preserve backwards compatibility with.